docs/api/rest_api/rest_api_reference/input/ez-product-products.raml

```diff
@@ -95,6 +95,163 @@
+                        body:
+                            application/json:
+                                type: Currency
+                                example: !include examples/product/catalog/product/code/prices/currencyCode/GET/Currency.json.example
+                    401:
+                        description: Error - the user has no permission to read prices.
+                    404:
+                        description: Error - the price does not exist.
+            /customer-group:
+                /{customerGroupIdentifier}:
+                    get:
+                        displayName: Get price in given currency and customer group
+                        description: Get price in given currency and customer group.
+                        headers:
+                            Accept:
+                                description: If set, gets price in given currency and customer group in JSON format.
+                                example:  |
+                                    application/vnd.ibexa.api.Price+json
+                            Content-Type:
+                                description: If set, the updated price is returned in JSON format.
+                                example:  |
+                                    application/json            
+                        responses:
+                            200:
+                                description: OK - gets price in given currency and customer group.
+                                body:
+                                    application/vnd.ibexa.api.Price+json:
+                                        type: Price
+                                        example: !include examples/product/catalog/product/code/prices/currencyCode/customer-group/customerGroupIdentifier/GET/PriceCurrencyCustomerGroup.json.example
+                            401:
+                                description: Error - the user has no permission to read prices.
+                            404:
+                                description: Error - the price does not exist.
+        /{id}:
+            delete:
+                displayName: Delete product price
+                description: Deletes product price.
+                headers:
+                    Accept:
+                        description: If set, deletes price in given currency in JSON format.
+                        example:  |
+                            application/vnd.ibexa.api.PriceDelete+json
+                    Content-Type:
+                        description: Delete product price.
+                        example:  |
+                            application/vnd.ibexa.api.PriceDelete+json
+                responses:
+                    200:
+                        description: No content - the given price is deleted.
+                    401:
+                        description: Error - the user is not authorized to delete this price.
+            patch:
```

Comment:
I'm missing the code of the product and id of the specific price being updated. The URL looks like: `products/{productCode}/prices/3`.

Reply:
@konradoboza 

ez.raml
```yaml
/product/catalog:
    /products: !include ez-product-products.raml
```
ez-product-products.raml
```yaml
/{code}:
    /prices:
        /{id}:
            patch:
```
 
 So in the reference, the concatenated route is `PATCH /product/catalog/products/{code}/prices/{id}`